docs/rst/user_manual/configuration.rst

```diff
@@ -93,6 +93,28 @@ topics and |ddsrouter| participants) are as big as to cause memory exhaustion is
 Likewise, one may choose to increase this value if wishing to deliver a greater number of samples to late joiners and
 enough memory is available.
 
+.. _user_manual_configuration_remove_unused_entities:
+
+Remove Unused Entities
+----------------------
+
+``specs`` supports a ``remove-unused-entities`` **optional** value that configures the deletion of unused internal entities in the |ddsrouter|.
+By default, unused internal entities are *not* removed.
+Thus, when the |ddsrouter| discovers a Subscriber, the |ddsrouter| creates entities in all of its participants, and these entities stay up even after the Subscriber disconnects.
+
+At times it can be useful to remove the internal entities that are not being used.
+Consider the following example.
+Two *DDS Routers* are communicating between each other through a WAN connection, when the last of their Subscribers disconnects.
```

Comment:
```suggestion
Two *DDS Routers* are communicating through a WAN connection, when the last of their Subscribers disconnects.
```

Reply:
The description is not clear in my opinion, it sounds as if it is the router's subscribers that disconnect. It is the external subscribers to which one of the routers is forwarding data.